dfp_external_storage/api.py

```diff
@@ -0,0 +1,79 @@
+from datetime import timedelta
+import frappe
+from frappe import _
+
+
+@frappe.whitelist()
+def generate_presigned_url(file_name, file_path="Record"):
```

Comment:
Why not using the `MinioConnection->presigned_get_object()` within `dfp_external_storage.py`?